packages/react-core/src/components/MultipleFileUpload/MultipleFileUploadButton.tsx

```diff
@@ -9,15 +9,20 @@ export interface MultipleFileUploadButtonProps extends React.HTMLProps<HTMLDivEl
   className?: string;
   /** Aria-label for the button */
   'aria-label'?: string;
+  /** Visible text label for the button */
   'label'?: string;
```

Comment:
Can we call this `text` or `browseButtonText` for consistency.  We call it `browseButtonText` in the `FileUpload` component.

Reply:
are you asking me to change the **'label' to `browseButtonText`?**